dataedit/templates/dataedit/dataedit_overview.html

```diff
@@ -109,6 +110,29 @@ <h3><a href="/dataedit/view/{{schema}}">{{schema}}</a>.{{table}} </h3>
     var data_record = {}
     var unchecked_record = {}
 
+
+
+    function addTooltips() {
+
+        var colname2tooltip = {}
+        // get description from metadata
+        {% for res in comment_on_table.resources %}
+            {% for field in res.fields %}
+                colname2tooltip["{{ field.name }}"] = "{{ field.description }}";
+            {% endfor %}
+        {% endfor %}
+
+        // don't use $, it's different version
+        jQuery(".slick-header-column").each(function(i, e){
+            var colDiv = jQuery(e);
+            var colName = colDiv.text();
+            var tooltipHTML = colname2tooltip[colName];
+            // TODO: html does not work
+            colDiv.attr('data-html', "true").attr('data-toggle', 'tooltip').attr('title', tooltipHTML);
+        });
+    };
+
+
     function build_views(){
```

Comment:
@wingechr - This function has apparently been replaced by `build_views_leaflet` and `build_views_recline`. I am not sure how to solve the merge conflict here